src/test/resources/feed-with-sitemap-namespace.xml

```diff
@@ -0,0 +1,238 @@
+<rss xmlns:dc="http://purl.org/dc/elements/1.1/" xmlns:atom="http://www.w3.org/2005/Atom" xmlns:sitemap="http://www.sitemaps.org/schemas/sitemap/0.9" xmlns:wordzilla="http://www.cadenhead.org/workbench/wordzilla/namespace" version="2.0">
```

Comment:
@silentninja, would you mind to replace the test document (feed-with-sitemap-namespace.xml) by a shorter one, preserving partially the structure but replacing the content by some dummy links and texts. The reason is that the Apache License also applies to test resources which is likely not the case for this feed. Thanks!